salt/modules/win_pkg.py

```diff
@@ -695,11 +696,13 @@ def install(name=None, refresh=False, pkgs=None, saltenv='base', **kwargs):
     new = list_pkgs()
     tries = 0
     difference = salt.utils.compare_dicts(old, new)
-    while not all(name in difference for name in changed) and tries <= 1000:
+    while not all(name in difference for name in changed) and tries < 10:
```

Comment:
I think this should be `tries <= 10` otherwise the `if tries == 10:` line below will never evaluate to `True`.